File: library_base/src/main/java/com/hjc/library_base/base/ILoadingView.java
package com.hjc.library_base.base;

/**
 * @Author: HJC
 * @Date: 2020/4/28 15:22
 * @Description: 自定义加载框需实现该接口
 */
public interface ILoadingView {

    /**
     * 显示loading
     */
    void showLoading();

    /**
     * 隐藏loading
     */
    void dismissLoading();

}

File: library-base/src/main/java/com/hjc/library_base/base/ILoadingView.kt
package com.hjc.library_base.base

/**
 * @Author: HJC
 * @Date: 2020/4/28 15:22
 * @Description: 自定义加载框需实现该接口
 */
interface ILoadingView {

    /**
     * 显示loading
     */
    fun showLoading()

    /**
     * 隐藏loading
     */
    fun dismissLoading()

}